Add missing translation for the Transfer error

amiskov opened the pull request:
## What issue does this PR close
Closes N/A. See [this message](https://cambiatus.slack.com/archives/CA83HJAAD/p1606953223006400) on Slack.

![image](https://user-images.githubusercontent.com/140053/100997225-14b34d80-356b-11eb-8ab3-36ea479c7d80.png)

## Changes Proposed ( a list of new changes introduced by this PR)
Add missing translations.

## How to test ( a list of instructions on how to test this PR)
Go to the Transfers page and try to send a large number of coins, like 70000. See the translated error.

lucca65 commented on public/translations/pt-BR.json:
```diff
@@ -111,7 +111,8 @@
         "Sale creator and sale doesn't match": "O criador da venda e a venda não correspondem",
-        "Invalid number of units": "Número inválido de unidades"
+        "Invalid number of units": "Número inválido de unidades",
+        "overdrawn community limit": "Limite de comunidade a descoberto"
```
```suggestion
        "overdrawn community limit": "Saldo insuficiente para realizar essa transferência"
```

lucca65 commented on public/translations/en-US.json:
```diff
@@ -111,7 +111,8 @@
         "Sale creator and sale doesn't match": "Sale creator and sale doesn't match",
-        "Invalid number of units": "Invalid number of units"
+        "Invalid number of units": "Invalid number of units",
+        "overdrawn community limit": "Overdrawn community limit"
```
```suggestion
        "overdrawn community limit": "Insufficient balance to make this transfer"
```

lucca65 commented on public/translations/es-ES.json:
```diff
@@ -111,7 +111,8 @@
         "Sale creator and sale doesn't match": "Creador de venta y venta no coincide",
-        "Invalid number of units": "Número de unidades no válido"
+        "Invalid number of units": "Número de unidades no válido",
+        "overdrawn community limit": "Límite comunitario sobregirado"
```
```suggestion
        "overdrawn community limit": "Saldo insuficiente para realizar esta transferencia"
```

lucca65 commented on public/translations/cat-CAT.json:
```diff
@@ -111,7 +111,8 @@
         "Sale creator and sale doesn't match": "El creador de la venda i la venda no coincideixen",
-        "Invalid number of units": "Nombre d'unitats no vàlid"
+        "Invalid number of units": "Nombre d'unitats no vàlid",
+        "overdrawn community limit": "Límit de comunitat superat"
```
```suggestion
        "overdrawn community limit": "Saldo insuficient per fer aquesta transferència"
```

lucca65 reviewed (changes requested):
The Amharic translation is also missing

`ይህንን ዝውውር ለማድረግ በቂ ያልሆነ ሚዛን`

amiskov commented:
> The Amharic translation is also missing

@lucca65 Thank you for noticing this and for improving all the errors! I noticed that there's no Amharic translation for contract errors at all. For now I just added one translation related to this PR and added additional item to the #419 issue for adding Amharic translations.

lucca65 commented:
I see! It took them a while to translate everything... we have more keys now. Sometimes I wish those json files were elm files so we could type check missing translations 🤔
